### PR TITLE
PIA-675: TVOS Compatibility 

### PR DESCRIPTION
**Pull request by kp-said-rehouni**
Summary

This PR adds compatibility to tvOS and removes all the UI code and resources to the main application in order to compile for iOS and tvOS.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -41,18 +41,18 @@ let package = Package(
...
                 .product(name: "PIACSI", package: "mobile-ios-releases-csi"),
                 .product(name: "PIARegions", package: "mobile-ios-releases-regions"),
                 .product(name: "PIAAccount", package: "mobile-ios-releases-account"),
                 .product(name: "PIAWireguard", package: "mobile-ios-wireguard"),
-                .product(name: "TunnelKit", package: "mobile-ios-openvpn"),
-                .product(name: "TunnelKitOpenVPN", package: "mobile-ios-openvpn"),
-                .product(name: "TunnelKitOpenVPNAppExtension", package: "mobile-ios-openvpn"),
+                .product(name: "TunnelKit", package: "mobile-ios-openvpn", condition: .when(platforms: [.iOS])),
+                .product(name: "TunnelKitOpenVPN", package: "mobile-ios-openvpn", condition: TargetDependencyCondition.when(platforms: [.iOS])),
+                .product(name: "TunnelKitOpenVPNAppExtension", package: "mobile-ios-openvpn", condition: TargetDependencyCondition.when(platforms: [.iOS]))
```
Comment (kp-juan-docal) [inline Package.swift:55]: I just noticed the condition. Does it mean we are not planning to support ovpn on tvOS?

**Review by kp-juan-docal — APPROVED** [review]
minor question. LGTM!

File: Package.swift (hunk 1 above)
Comment (kp-said-rehouni) [reply Package.swift:55]: Good catch! Not for now. There were still some issues with OpenSSL on tvOS that we will address on the future. For now let's make tvOS work and connect with IKEV2, then we can fix OpenVPN.
Thank you @kp-juan-docal!

**Review by kp-juan-docal — DISMISSED** [review]

**Comment by kp-juan-docal** [comment]
@kp-said-rehouni we are bringing over 121 commits that are not necessarily about TV compatibility. Is that intentional?

File: Package.swift [hunk 2]
```diff
@@ -41,18 +41,18 @@ let package = Package(
...
-                "PopupDialog",
+                .product(name: "PopupDialog", package: "PopupDialog", condition: .when(platforms: [.iOS])),
+                .product(name: "SwiftEntryKit", package: "SwiftEntryKit", condition: .when(platforms: [.iOS])),
                 .product(name: "Lottie", package: "lottie-ios"),
                 .product(name: "Reachability", package: "Reachability.swift"),
                 .product(name: "PIAKPI", package: "mobile-ios-releases-kpi"),
                 .product(name: "PIACSI", package: "mobile-ios-releases-csi"),
                 .product(name: "PIARegions", package: "mobile-ios-releases-regions"),
                 .product(name: "PIAAccount", package: "mobile-ios-releases-account"),
                 .product(name: "PIAWireguard", package: "mobile-ios-wireguard"),
```
Comment (kp-laura-sempere) [inline Package.swift:52]: @kp-said-rehouni , we need to do the same with WG, right? We also need to only include it on iOS platform atm

Comment (kp-said-rehouni) [reply Package.swift:52]: Right! Thanks for the reminder @kp-laura-sempere. 
Updating it!

Comment (kp-said-rehouni) [reply Package.swift:52]: After checking the impact, it's probably better to do it on a different PR. PIAWireguard contains some dependencies that PIALibrary uses, such as Alamofire. We need to add also those dependencies to PIALibrary and add availability checks for all the classes that import PIAWireguard. What do you think @kp-laura-sempere?

Comment (kp-laura-sempere) [reply Package.swift:52]: No problem Said, I can also do this update on the WG crash ticket that I'm working on
I already had to move the Alamofire dependency to PIALibrary in the branch I'm working on

**Review by kp-laura-sempere — APPROVED** [review]
Great job! 👍

File: Package.swift (hunk 2 above)
Comment (kp-said-rehouni) [reply Package.swift:52]: Thank you @kp-laura-sempere!

**Review by kp-laura-sempere — APPROVED** [review]
Great job 🚀

**Pull request merged**